src/store/modules/principal.js

```diff
@@ -21,7 +35,8 @@ const state = {
   activeResource: null,
   resourceDialogIsOpen: false,
   confirmationDialogIsOpen: false,
-  resourceViewDialogIsOpen: false
+  resourceViewDialogIsOpen: false,
+  statuses: {'IN_MISSION': 2, 'AVAILABLE': 1, 'UNAVAILABLE': 0}
```

Comment:
Consider using the enums from the contract instead of hardcoded strings for  IN_MISSION , AVAILABLE, UNAVAILABLE